python/docs/source/getting_started/install.rst

```diff
@@ -131,15 +131,15 @@ to install Spark, for example, as below:
 
 .. code-block:: bash
 
-    tar xzvf spark-3.4.0-bin-hadoop3.tgz
+    tar xzvf spark-3.5.0-bin-hadoop3.tgz
```

Comment:
It seems that sphinx-doc also supports variable substitution

Reply:
I wasn't able to find the usage patter in Apache Spark repo. Could you provide a link?

Reply:
I actually have an example:

- https://github.com/apache/spark/blob/master/python/docs/source/conf.py#L86C1-L99
- https://github.com/apache/spark/blob/9e41de2b1f23fc57425c8327b1bf88398e597fe5/python/docs/source/getting_started/index.rst#L30

Reply:
I can follow up @dongjoon-hyun. Please go ahead and merge if that's preferred.

Reply:
You can use the `release` variable defined in the config file. FYI, https://github.com/apache/spark/blob/3830285d56d8971c6948763aeb3d99c5bf5eca91/python/docs/source/conf.py#L128

It is inlined inside a code block, so maybe you also need the `parsed-literal` directive to resolve it ahead.

Reply:
FYI, an example from apache/kyuubi, https://github.com/apache/kyuubi/blob/master/docs/quick_start/quick_start.rst#install-kyuubi